src/client/components/input/partial-date.js

```diff
@@ -53,10 +70,18 @@ class PartialDate extends React.Component {
 		this.handleChange = this.handleChange.bind(this);
 	}
 
+	/**
+	* To get the value filled in the form field.
+	* @returns {string} - Value filled in the form field.
+	*/
 	getValue() {
 		return this.state.value;
 	}
 
+	/**
+	* Track the changes in the date field of the form and validate the value in
```

Comment:
Hmm, this doesn't really track anything, strictly speaking. It's "An event handler, which is triggered whenever the value in the date field changes". And it "validates the value... and updates the components state with the new value and validation state". It also calls the component's `onChange` handler, if this is set.